### ISSUE TITLE
[Bug]: (I suppose?) Automatic1111: Moving cursor within the canvas = black screen

### ISSUE THREAD

**Issue by herman925** [zero01101/openOutpaint-webUI-extension#43]
### Is ths issue about the extension?

- [X] Yup, this is for sure about the extension

### What happened?

![image](https://user-images.githubusercontent.com/65190774/223091159-42fc16c7-cdaa-4a62-9570-20d4a3d18256.png)

### Steps to reproduce the problem

Just going to the tab, move the cursor within the canvas, and the screen will turn black.

### What should have happened?

I suppose the canvas should've not been black but instead showing the checkered board pattern indicating transparency?

### Commit where the problem happens

bf21c19

### What platforms do you use to access openOutpaint?

Windows

### What browsers do you use to access the UI ?

Microsoft Edge

### Browser Extensions/Addons

openOutpaint-webUI-extension
posex
sd-webui-additional-networks
sd-webui-ar
sd-webui-controlnet

![image](https://user-images.githubusercontent.com/65190774/223091580-0500531c-8c37-4f2a-a032-c74aa69698e5.png)

### AUTOMATIC1111 webUI Commandline Arguments

--xformers --medvram --disable-safe-unpickle --opt-split-attention --api --no-half-vae

**Comment by zero01101** [comment]
hey there; so the "Browser Extensions/Addons" section was actually about your web browser's extensions, not webUI lol - we ask that because [the duckduckgo privacy extension broke openOutpaint pretty good](https://github.com/zero01101/openOutpaint-webUI-extension/issues/3#issuecomment-1367694000) and i kind of suspect an extension might be to blame here, as openOutpaint should otherwise definitely work on edge...

other than the browser addons/extensions, do you possibly have a different browser you could test with to see if it occurs there as well?

**Comment by herman925** [comment]
Hmm, it's quite weird because I didn't install any extension on Edge. It's a plain Edge (to help make sure I have enough memory for Automatic1111).

I tried it on another Chromium browser with all the extensions however and it worked there. So I'm not sure why. Is there anything I can provide to further investigate into this?

**Comment by zero01101** [comment]
yeah, i'm 100% stymied on this one as it works as expected on basic, almost never used for anything edge v111.0.1661.41... if i can come up with anything you could check i'll be sure to update lol, but otherwise if VRAM is your concern, you could potentially disable your browser of choice from using hardware acceleration (at least in chrome, edge, and firefox, so i presume it's pretty universal) but i 100% think that makes openOutpaint, like, completely unusable :(

**Comment by Walhalla-Summary** [comment]
File "F:\AIPRO\lq_drew\drew\stable-diffusion-webui\extensions\openOutpaint-webUI-extension\scripts\main.py", line 55, in update_app
    run(f'"{git}" -C "' + os.path.join(scripts.basedir(), usefulDirs[0], usefulDirs[1]) +
  File "F:\AIPRO\lq_drew\drew\stable-diffusion-webui\launch.py", line 89, in run
    raise RuntimeError(f"""{errdesc or 'Error running command'}.
RuntimeError: Error running command.
Command: "git" -C "F:\AIPRO\lq_drew\drew\stable-diffusion-webui\extensions\openOutpaint-webUI-extension" submodule update --init --recursive --remote
Error code: 1

During handling of the above exception, another exception occurred:



@zero01101

**Comment by zero01101** [comment]
@Walhalla-Summary don't think that's at all related to what was being experienced in the initial issue, but that suggests that python wasn't able to run git - do you have it installed, available in PATH, etc?
